wallet/src/de/schildbach/wallet/ui/dashpay/PlatformRepo.kt

```diff
@@ -277,7 +333,7 @@ class PlatformRepo(val walletApplication: WalletApplication) {
     //
     suspend fun verifyIdentityRegisteredAsync(blockchainIdentity: BlockchainIdentity) {
         withContext(Dispatchers.IO) {
-            blockchainIdentity.watchIdentity(10, 5000, BlockchainIdentity.RetryDelayType.SLOW20)
+            blockchainIdentity.watchIdentity(10, 5000, RetryDelayType.SLOW20)
```

Comment:
Travis CI failed due to `RetryDelayType` being an undefined symbol.

Reply:
I see that it was moved in `dashpay`.  That explains that.

Reply:
It was moved to be reused, yes.